x-pack/plugins/observability_solution/logs_explorer/public/components/common/translations.tsx

```diff
@@ -227,8 +87,18 @@ export const contentHeaderTooltipParagraph1 = (
     id="xpack.logsExplorer.dataTable.header.content.tooltip.paragraph1"
     defaultMessage="Displays the document's {logLevel} and {message} fields."
     values={{
-      logLevel: <strong>log.level</strong>,
-      message: <strong>message</strong>,
+      logLevel: (
+        <strong>
+          {i18n.translate('xpack.logsExplorer..strong.loglevelLabel', {
```

Comment:
Should these really be translated? If they should, there's a duplicate `.` in the id.

Reply:
They shouldn't, I didn't event noticed that because the linter auto fix it due to the recent rule `@kbn/i18n/strings_should_be_translated_with_i18n`. I'll change it back.